flexmeasures/data/migrations/versions/6cf5b241b85f_copy_attributes_from_old_data_models_to_GenericAsset.py

```diff
@@ -21,7 +23,13 @@
 
 def upgrade():
     op.add_column(
-        "generic_asset", sa.Column("attributes", sa.JSON(), nullable=True, default="{}")
+        "generic_asset",
+        sa.Column(
+            "attributes",
+            MutableDict.as_mutable(postgresql.JSONB(astext_type=sa.Text())),
```

Comment:
Please explain why this is chosen over `JSON` (in a comment).

Reply:
As discussed on our chat, posted here for future reference: the postgres version comes with some [extra operators](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#sqlalchemy.dialects.postgresql.JSONB) to filter on keys within the JSON. But since we're not using these at the moment and are still unsure whether we will need them in the future, we prefer to keep this new column agnostic to the database type. Therefore, I moved it back to a regular `sqlalchemy.types.JSON` (i.e. `db.JSON`), while retaining its mutability.